src/app/admin/mod.rs

```diff
@@ -43,6 +42,10 @@ where
         }
     }
 
+    pub fn into_accept(self) -> Accept<M> {
```

Comment:
Maybe put this into an `Into<Accept<M>>`?

Reply:
that will still be awkward... the users of this don't actually care about things being an `Accept`, they care that it's an `impl Accept...` So you'd have to do some weird explicit type specification..